Rivet/Functions/Repair-Operation.ps1

```diff
@@ -0,0 +1,136 @@
+            'AddPrimaryKeyOperation'
+            {
+                $name = New-ConstraintName -PrimaryKey -SchemaName $schemaName -TableName $tableName
+                $warningMsg = "Primary key constraint names will be required in a future version of Rivet. Please " +
+                              "add a ""Name"" parameter (with a value of ""$($name)"") to the Add-PrimaryKey " +
+                              "operation for the $($tableDesc) table's $($columnDesc) column."
+            }
+            'AddUniqueKeyOperation'
+            {
+                $name = New-ConstraintName -UniqueKey -SchemaName $schemaName -TableName $tableName -ColumnName $columnName
+                $warningMsg = "Unique key constraint names will be required in a future version of Rivet. Please add " +
+                              "a ""Name"" parameter (with a value of ""$($name)"") to the Add-UniqueKey " +
+                              "operation on the $($tableDesc) table's $($columnDesc) column$($pluralSuffix)."
+            }
+            'RemoveDefaultConstraint'
+            {
+                $name = New-ConstraintName -Default -SchemaName $schemaName -TableName $tableName -ColumnName $columnName
+                $warningMsg = "Default constraint names will be required in a future version of Rivet. Please add a " +
+                              """Name"" parameter (with a value of ""$($name)"") to the Remove-DefaultConstraint " +
+                              "operation for the $($tableDesc) table's ""$($columnDesc)"" column."
+            }
+            'RemoveForeignKeyOperation'
+            {
+                $name = New-ConstraintName -ForeignKey 
+                                           -SchemaName $schemaName `
+                                           -TableName $tableName `
+                                           -ReferencesSchema $Operation.ReferencesSchema `
+                                           -ReferencesTableName $Operation.ReferencesTableName
+                $warningMsg = "Foreign key constraint names will be required in a future version of Rivet. Please " +
+                              "add a ""Name"" parameter (with a value of ""$($name)"") to the Remove-ForeignKey " +
+                              "operation for the $($tableDesc) table that references the " +
+                              "[$($Operation.ReferencesSchemaName)].[$($Operation.ReferencesTableName)] table."
+            }
+            'RemoveIndexOperation'
+            {
+                $name = New-ConstraintName -Index -SchemaName $schemaName -TableName $tableName -ColumnName $columnName -Unique:$Operation.Unique
+                $warningMsg = "Index names will be required in a future version of Rivet. Please add a ""Name"" " +
+                              "parameter (with a value of ""$($name)"") to the Remove-Index operation for the " +
+                              "$($tableDesc) table's ""$($columnDesc)"" column$($pluralSuffix)."

```

Comment:
Different formatting (and a few others).

Reply:
Again, I don't see what's different.